src/var/www/hsmm-pi/View/NetworkServices/index.ctp

```diff
@@ -4,22 +4,22 @@
 </div>
 
 <div class="row">
-  <div class="span12">
+  <div class="col-md-12">
     <span class="pull-right">
-      <p>
+      <div>
```

Comment:
I meant: change the `<span>` to a `<div>` and keep the `<p>`. `<div>` isn't allowed inside `<span>`.